crates/router/src/core/payments/operations/payment_complete_authorize.rs

```diff
@@ -247,6 +249,14 @@ impl<F: Send + Clone, Ctx: PaymentMethodRetrieve>
                 id: profile_id.to_string(),
             })?;
 
+        let recurring_details = request
```

Comment:
Why are we ignoring `MandateId` here?

Reply:
because in `PaymentsRequest`, we have both `mandate_id: Option<String>` and `recurring_details: Option<String>` where recurring_details is mapped to payment method id